docker-compose.yml

```
@@ -0,0 +1,12 @@
+version: '3'
+
+services:
+  app:
+    build: .
+    ports:
+      - "3000:3000"
+    volumes:
+      - .:/app
+      - /app/node_modules
+    environment:
+      - .env.local
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)